### PR TITLE
License new

### LINKED ISSUE #22: Also add license
Github's API provides machine-readable license information that can be used to add that information to the Wikidata item in case it is missing.

### PR DESCRIPTION
**Pull request by Nudin**
Replace  #38, fixes #22.

I also added types to two functions and renamed a variable to avoid confusion.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -60,6 +63,7 @@ class Properties:
     source_code_repository = "P1324"
     title = "P1476"
     protocol = "P2700"
+    license = "P275"
```
Comment (konstin) [inline main.py]: Apparently there's a builtin function `license()`. Not sure if that actually matters here though

File: main.py [hunk 2]
```diff
@@ -39,6 +39,9 @@ class Settings:
     whitelist: List[str] = []
     sparql_file = "free_software_items.rq"
 
+    license_sparql_file = "free_licenses.rq"
```
Comment (konstin) [inline main.py]: That file needs to be added

Comment (konstin) [inline main.py]: That file needs to be added

File: main.py (hunk 1 above)
Comment (Nudin) [reply main.py]: Get's overwritten and is not needed for anything (just shows the license of the python interpreter)

File: main.py (hunk 2 above)
Comment (Nudin) [reply main.py]: done

**Comment by konstin** [comment]
Thank you!

**Pull request merged**